AllReadyApp/Web-App/AllReady.UnitTest/Areas/Admin/Controllers/SiteAdminControllerShould.cs

```diff
@@ -156,14 +158,34 @@ public void EditUserGetReturnsCorrectViewModelWhenOrganizationIdIsNull()
             }
         }
 
-        [Fact(Skip = "NotImplemented")]
+        [Fact]
         public void EditUserGetReturnsCorrectViewModelWhenOrganizationIdIsNotNull()
         {
+            var mediator = new Mock<IMediator>();
+            var user = new ApplicationUser();
+            int orgId = 99;
+            string orgName = "Test Org";
+            mediator.Setup(x => x.Send(It.Is<UserByUserIdQuery>(q => q.UserId == userId)))
+                            .Returns(user);
+
+            mediator.Setup(x => x.Send(It.Is<OrganizationByIdQuery>(q => q.OrganizationId == orgId)))
+                            .Returns(org);
+
+            var controller = new SiteController(null, null, mediator.Object);
+
+            var result = controller.EditUser(userId);
+            var model = ((ViewResult)result).ViewData.Model as EditUserModel;
+
+            Assert.NotNull(model.Organization);
+            Assert.IsType<EditUserModel>(model);
+            Assert.Equal(model.Organization.Id, orgId);
+            Assert.Equal(model.Organization.Name, orgName);
```

Comment:
This looks good. Since the only property on EditUserModel that is directly affected by the OrganizationId being null is the Organization property, you could just assert that model.Organization is not null. The other two assertions (model.Organization.Id and model.Organization.Name) can be removed. You can probably rename this test to:
`EditUserGetReturnsCorrectValueForOrganiztionOnViewModelWhenOrganizationIdIsNotNull()`

Some other ideas for missing unit tests for this action method are:
- EditUserGetReturnsCorrectValueForIsOrganizationAdminOnViewModelWhenUserIsOrgAdmin()
- EditUserGetReturnsCorrectValueForIsSiteAdminOnViewModelWhenUserIsSiteAdmin()
(and write opposite tests for each of these tests)